lib/shared/log/tasks.js

```diff
@@ -13,7 +13,23 @@ function isObject(val) {
   return typeof val === 'object' && !Array.isArray(val);
 }
 
+function wrapGetTask(getTask) {
+  return function(name) {
+    return createTask(getTask(name));
+  };
+}
+
+function createTask(task) {
+  var unwrapped = task.unwrap ? task.unwrap() : {};
```

Comment:
1. try to read `task.description` before unwrapping
2. if it is empty, check for `typeof task.unwrap === 'function'`, unwrap and check for `unwrapped.description`